en/reference/basic-mapping.rst

```diff
@@ -135,8 +135,11 @@ See `PHPCR\\PropertyType <http://phpcr.github.io/doc/html/files/phpcr.src.PHPCR.
 - ``Name``: A valid PHPCR name
 - ``Path``: A valid PHPCR path
 - ``Uri``: A valid URI, for example a URL
-- ``Uuid``: A PHPCR universally unique id for a document (read only, see :ref:`References <association-mapping_referenceable>` for more)
```

Comment:
uuid is not mapped as field. this lead to the confusion in https://github.com/doctrine/phpcr-odm/issues/697